clickhouse-core/src/main/scala/com/clickhouse/spark/client/NodeClient.scala

```diff
@@ -53,7 +54,6 @@ class NodeClient(val nodeSpec: NodeSpec) extends AutoCloseable with Logging {
       "Spark-ClickHouse-Connector"
     }
   }
-
```

Comment:
unnecessary change, and please keep one empty line between functions/values that ha multi lines defination